blocks/feed/feed.js

```diff
@@ -63,6 +67,19 @@ export async function fetchBlogContent(url) {
       newHeading.innerHTML = heading.innerHTML;
       heading.replaceWith(newHeading);
     });
+    // render images as img tags rather than linka
+    doc.querySelectorAll('a').forEach((link) => {
+      const linkSrc = new URL(link.href);
+      const { pathname } = linkSrc;
+      if (isImgUrl(pathname)) {
+        const imgName = pathname.substring(pathname.lastIndexOf('/') + 1, pathname.lastIndexOf('.'));
+        const img = createTag('img', {
```

Comment:
would we want to use createOptimizedPicture here instead?

Reply:
also does it need to be eager? what if the image is not above the fold?

Reply:
addressed the review comments